.github/workflows/no_old_refs.yml

```diff
@@ -0,0 +1,31 @@
+#####################################################################################
+# Github Action to check references to closed issues in code.
+#
+# Workflow starts when:
+# 1) push on master branch
+#
+#####################################################################################
+name: "Check no closed issue references"
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  check_issues:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download checkstyle
+        uses: actions/checkout@v2
+        with:
+          repository: checkstyle/checkstyle
+          ref: master
+          path: checkstyle
+
+      - name: Check Issues
+        env:
+          GITHUB_TOKEN: '${{ secrets.GITHUB_TOKEN }}'
+        run: |
+          cd checkstyle
+          chmod +x ./.ci/no_old_refs.sh
```

Comment:
please make this change on your local and commit file with this execution attribute, strange that 
https://github.com/checkstyle/checkstyle/blob/master/.ci/travis/checkchmod.sh
is not reporting this problem.

Reply:
````
HEAD is now at 18a37979b Issue #9152: Record implementing interfaces, indentation faulty
-rwxr-xr-x  1 www-data www-data  1060 Feb 21 23:04 no_old_refs.sh*
````

Looks like executable is not an issue. He may not originally had it and committed it after a run setting it.

Reply:
done, I removed this line, looks like it is ok without chmod